LICENSE.txt

```diff
@@ -16,3 +16,30 @@ FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT,
 NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION
 WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
+
+
+Support for epytext and reStructuredText was imported from epydoc.
+This code can be found under pydoctor/epydoc/ and is licensed as follows:
```

Comment:
Should this say "originally licensed as follows" rather than "is licensed as follows"? The above license is what applies to the combined work and the modifications to the original made in this repository, no?

Reply:
"Originally licensed" might suggest the code was re-licensed, which is not the case. The original code is still under the original license, modifications are under pydoctor's license. But most of the changes I made to the epydoc code are removing features that pydoctor doesn't need and replacing intermediate steps, plus a few bug fixes. So there is not any new feature or cluster of code that stands by itself, it's small modifications in many places.